src/phaser-game/scenes/FortNerf.ts

```diff
@@ -1,168 +1,198 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-shadow */
 import Phaser from 'phaser';
+import { BULLET_MOVEMENT, BULLET_OFFSET } from '../utils/constants';
 
-// let platforms;
-let map: any;
+let background: any;
 let player: any;
-let cursors: any;
-// let score = 0;
-// let usernameText: any;
-
+let upCursor: any;
+let downCursor: any;
+let leftCursor: any;
+let rightCursor: any;
+let spaceBar: any;
+let walls: any;
+let shootBullet: any;
+let bullet: any = {};
 class FortNerf extends Phaser.Scene {
   constructor() {
     super('FortNerf');
   }
 
   preload() {
-    this.load.image('background', '/assets/pallet_town.jpg');
+    this.load.image('background', '/assets/sky.png');
```

Comment:
Maybe consider building all of these image asset strings into an object and referencing them by key name here to aggregate that all into one place and cleanup the readability of this section a bit?